packages/remix-dev/compiler/plugins/serverNativeNodeModules.ts

```diff
@@ -0,0 +1,32 @@
+import type { Plugin } from "esbuild";
+
+// See https://github.com/evanw/esbuild/issues/1051#issuecomment-806325487
+export const serverNativeNodeModulesPlugin: Plugin = {
```

Comment:
Let's make this a function, so it's consistent with our other plugins

```suggestion
export const serverNativeNodeModulesPlugin = (): Plugin => ({
```